spec/values/calculation/calc/operator.hrx

```diff
@@ -28,14 +28,54 @@ a {
 
 <===>
 ================================================================================
-<===> minus/preserved/input.scss
+<===> minus/preserved/number/input.scss
 a {b: calc(1px - 2%)}
 
-<===> minus/preserved/output.css
+<===> minus/preserved/number/output.css
 a {
   b: calc(1px - 2%);
 }
 
+<===>
+================================================================================
+<===> minus/preserved/multiplication/input.scss
+a {b: calc(1px - (2% * var(--c)))}
+
+<===> minus/preserved/multiplication/output.css
+a {
+  b: calc(1px - 2% * var(--c));
+}
+
+<===>
+================================================================================
+<===> minus/preserved/division/input.scss
+a {b: calc(1px - (2% / var(--c)))}
+
+<===> minus/preserved/division/output.css
+a {
+  b: calc(1px - 2% / var(--c));
+}
+
+<===>
+================================================================================
+<===> minus/preserved/plus/input.scss
+a {b: calc(1px - (2% + var(--c)))}
+
+<===> minus/preserved/plus/output.css
+a {
+  b: calc(1px - (2% + var(--c)));
+}
+
+<===>
+================================================================================
+<===> minus/preserved/minus/input.scss
+a {b: calc(1px - (2% - var(--c)))}
+
+<===> minus/preserved/minus/output.css
+a {
+  b: calc(1px - (2% - var(--c)));
+}
+
```

Comment:
Would it make sense to add similar tests for plus/times/divide?

Reply:
We have them for multiplication and division below under "precedence", but I suppose it couldn't hurt to add them for plus as well.